src/utils/azure.ts

```diff
@@ -72,30 +74,31 @@ export async function getStorageConnectionString(context: IStorageAccountWizardC
 
 export async function getEventHubsConnectionString(context: IEventHubsConnectionWizardContext & ISubscriptionContext): Promise<IResourceResult> {
```

Comment:
Could you move all these `get____ConnectionString` functions out of `azure.ts`? I think it would make sense if they were in `appSettings/connectionSettings` as its own file for getting connection strings.

Reply:
Trueee 🙌